pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/EtcdMetadataStore.java

```diff
@@ -285,7 +285,7 @@ protected void batchOperation(List<MetadataOp> ops) {
                                 .withKeysOnly(true)
                                 .withSortField(GetOption.SortTarget.KEY)
                                 .withSortOrder(GetOption.SortOrder.ASCEND)
-                                .withPrefix(prefix)
+                                        .isPrefix(true)
```

Comment:
Please don't change the indent.

Reply:
This is the auto format. and CI is green. Maybe i missed something? 

@BewareMyPower PTAL,Thanks.

Reply:
I just got it and updated.